.ci/rebuild_updated_recipes.py

```diff
@@ -16,12 +16,14 @@ def modified_recipes(branch='origin/master'):
     for file_path in git_diff:
         if 'recipes/' in file_path:
             recipe = file_path.split('/')[1]
-            recipes.add(recipe)
+            if recipe not in ["python", "hostpython"]:  # alias recipes
+                recipes.add(recipe)
```

Comment:
did you sneak a bugfix in this linter PR?

Reply:
Well, they  seem redundant? They are alias recipes for `python2` or `python3' and `hostpyhton2` and `hostpython3` respectively. So the actual recipes are checked, it 's just the alias that does not exists. From what I understand anyway.

Reply:
I haven't dug into it, but the CI is young enough and we would take responsibility to revert or fix if we introduced a bug here. So I'm OK that we take the risk

Reply:
BTW touching it I understood what you were doing. I misread this:
```
if recipe not in ["python", "hostpython"]:
```
For that:
```
if recipe in recipes:
```
Sorry about that :facepalm: